proxy/tracing.go

```diff
@@ -12,7 +12,7 @@ const (
 	HostnameTag           = "hostname"
 	HTTPHostTag           = "http.host"
 	HTTPMethodTag         = "http.method"
-	HTTPRemoteAddrTag     = "http.remote_addr"
+	HTTPRemoteAddrTag     = "http.remote_addr" // unused
```

Comment:
Maybe add a TODO: v1 cleanup or we just remove the const, IMO it's not really "public interface"

Reply:
yep, looks like they were made public unintentionally by #1066

Reply:
Removed this and another one unused since #1907